src/component/ResponsiveContainer.tsx

```diff
@@ -27,6 +27,7 @@ export interface Props {
     width: number;
     height: number;
   };
+  'data-testid'?: string;
```

Comment:
What if someones test key is different than `data-testid`? could we allow all data-* attributes? 

 http://www.typescriptlang.org/docs/handbook/jsx.html#attribute-type-checking
"Note: If an attribute name is not a valid JS identifier (like a data-* attribute), it is not considered to be an error"

So data-* attributes are implicitly supported by TS, we can remove the explicit addition.

we could have a function called `getBaseProps` or similar that looks for props that match data-* and spreads them on the components.